mmv1/products/accesscontextmanager/terraform.yaml

```diff
@@ -157,6 +157,12 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         vars:
           access_level_name: "chromeos_no_lock"
           service_perimeter_name: "restrict_storage"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "access_context_manager_service_perimeter_secure_data_exchange"
+        skip_test: true
```

Comment:
Why skip test here?

Reply:
Well, these are all marked as skip test because they are only for example generation. Can we add a test for this to the handwritten test file so our CI will run them regularly?

I believe it is this file: https://github.com/GoogleCloudPlatform/magic-modules/blob/master/mmv1/third_party/terraform/tests/resource_access_context_manager_service_perimeter_test.go.erb